src/fragments/orders.ts

```diff
@@ -154,6 +154,12 @@ export const fragmentOrderLine = gql`
 export const fragmentOrderLineWithMetadata = gql`
   fragment OrderLineWithMetadata on OrderLine {
     ...OrderLine
+    metadata {
+      ...MetadataItem
+    }
+    privateMetadata {
+      ...MetadataItem
+    }
     variant {
```

Comment:
Suggestion: We could just use `Metadata` fragment from `src/fragments/metadata.ts`